packages/dev/test-utils/package.json

```diff
@@ -25,8 +25,8 @@
     "@react-spectrum/theme-default": "^3.5.13",
     "@swc/helpers": "^0.5.0",
     "@testing-library/dom": "^10.1.0",
-    "@testing-library/jest-dom": "^5.16.4",
-    "@testing-library/react": "^15.0.7",
+    "@testing-library/jest-dom": "^6.0.0",
+    "@testing-library/react": "^16.0.0",
```

Comment:
Doesn't seem to have any breaking changes with this bump and the tests ran clean but will need to confirm on a React 19 project that the test utils work now